src/CommunityToolkit.Maui.UnitTests/Extensions/TextColorTo_Tests.cs

```diff
@@ -115,4 +143,37 @@ interface ICustomTextStyle : ITextStyle
 	{
 
 	}
+
+	public interface ISomeInterface
+	{
+
+	}
+	public class ClassConstraintWithInterface : ISomeInterface
+	{
+
+	}
+	public class ClassConstraint
+	{
+
+	}
+
+	public struct StructConstraint
+	{
+
+	}
+	public class GenericPicker<TA, TB, TC, TD, TE, TF, TG, TH, TI, TJ, TK> : Picker
```

Comment:
Will this work with records? Can you add them to the test case?

Reply:
@pictos yes I do have a case scenario. I'm porting existing solution from `Xamarin.Forms` to `MAUI` and it has `public class EnumPicker<T> : Picker where T : Enum {}` where body of the class populates `ItemsSource` via all possible `Enum` values and takes displayed names from `DisplayAttribute`. I wasn't really interested in this particular color extension, but rather wanted to drop support of custom `IAsyncCommand` in the project in favor of `IAsyncRelayCommand` from this library. After installing the nuget my project stopped building with error mentioned above until I implemented the changes.